parmed/amber/netcdffiles.py

```diff
@@ -264,7 +264,7 @@ def coordinates(self, stuff):
     def velocities(self):
         if 'velocities' in self._ncfile.variables:
             vels = self._ncfile.variables['velocities'][:]
-            return (vels.reshape((-1, self.atom, 3)) * self.velocity_scale)
+            return (vels.reshape((1, self.atom, 3)) * self.velocity_scale)
```

Comment:
This should be `-1` because there may be more than 1 frame.  The `-1` indicates that the dimension should take whatever value necessary to give a shape that accounts for the full length of the array.  For NetCDF trajectories that have more than 1 frame, this will fail.

Reply:
@swails , makes sense. Thank you for carefully checking on this.

`pyflakes` is detecting failures and for this reason it is killing the Travis CI build. What should I do to fix this? Just remove the "imported but unused" stuff?

Thanks!

Reply:
Yes, resolve the pyflakes issues.

Reply:
@swails : continuing on the first comment you made, I noticed that a few lines above ( parmed/amber/netcdffiles.py#255 ) there is an equivalent function for the coordinates, and there is no `-1` inside the `reshape` function:
```
            return coords.reshape((1, self.atom, 3))
```
This is why I made the change you commented in the `velocities` function. Shall I place a `-1` in this line as well?

Reply:
Yes, I think that's a good idea.